### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_movies-list-filter/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [film, findFilm] = useState('');
+
+  const moviesFilter = moviesFromServer.filter((movie) => {
+    const { description, title } = movie;
```
Comment (rashyd-hasratov) [inline src/App.tsx:10]: You also can use destruction in params
```suggestion
  const moviesFilter = moviesFromServer.filter(({ title, description }) => {
```

File: src/App.tsx [hunk 2]
```diff
@@ -20,12 +30,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={film}
```
Comment (rashyd-hasratov) [inline src/App.tsx:33]: I'm not sure you need this line of code

**Review by rashyd-hasratov — APPROVED** [review]
Well done, just check my suggestions please

File: src/App.tsx [hunk 3]
```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [film, findFilm] = useState('');
```
Comment (sergii-nosachenko) [inline src/App.tsx:7]: https://react.dev/reference/react/useState#usestate

File: src/App.tsx [hunk 4]
```diff
@@ -20,12 +30,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={film}
+                onChange={(event) => findFilm(event.target.value)}
```
Comment (sergii-nosachenko) [inline src/App.tsx:34]: Create handler for this

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**